client-templates/bitbucket-server/accept.json.sample

```diff
@@ -628,18 +637,18 @@
     {
       "//": "used to create a Code Insights report",
       "method": "PUT",
-      "path": "/rest/insights/1.0/projects/:project/repos/:repo/commits/:sha/reports/:report",
+      "path": "/rest/insights/:apiVersion/projects/:project/repos/:repo/commits/:sha/reports/:report",
```

Comment:
Replace `1.0` by `:apiVersion` for compatibility with Bitbucket API

Reply:
A)
where does `apiVersion` come from? is it "BBS api version" or "code insights" version (which I think it was meant to be)?

Reply:
B)
do we also need to make the change in the code?
https://github.com/snyk/bitbucket-server-agent/blob/develop/lib/bitbucket-server/codeinsights.js#L109
https://github.com/snyk/bitbucket-server-agent/blob/develop/lib/bitbucket-server/codeinsights.js#L129

Reply:
`:apiVersion` is url template variable that will allow to pass any API version both `/1.0` or `/latest`